kategory-docs/typeclasses/applicative/README.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: docs
+title: Applicative
+permalink: /docs/typeclasses/applicative/
+---
+
+## Functor
+
+The `Applicative` typeclass abstracts the ability to lift values and apply functions over the computational context of a type constructor.
+Examples of type constructors that can implement instances of the Applicative typeclass include `Option`, `NonEmptyList`,
+`List` and many other datatypes that include a `pure` and either `ap` function. `ap` may be derived for monadic types that include a `Monad` instance via `flatMap`.
+
+### Applicative Builder examples
+
+Often times we find ourselves in situations where we need to compute multiple independent values resulting from operations that do not depend on each other.
+
+In the following example we will define 3 invocations that may as well be remote or local services each one of them returning different results in the same computational context of `Option`
+
+```kotlin:ank
```

Comment:
so is `ank` parsing this snippets and compiling them to fail CI if docs snippets are not compiling for any reason ?